plugin-server/tests/helpers/metrics.ts

```diff
@@ -0,0 +1,15 @@
+import { register } from 'prom-client'
+
+export function resetMetrics() {
+    register.resetMetrics()
+}
+
+export async function getMetricValues(metricName: string): Promise<any> {
```

Comment:
fwiw we do a similar thing in the functional tests, which would have caught the piscina separate process issue. I need to make these easier to run though :-/